src/etc/inc/notices.inc

```diff
@@ -461,10 +486,67 @@ function curl_post_telegram($url) {
 	return $curl_post_telegram_data;	//json encoded
 }
 
+/****f* notices/notify_via_pushover
+ * NAME
+ *   notify_via_pushover
+ * INPUTS
+ *	 notification string to send to Pushover via API
+ * RESULT
+ *   returns NULL if message was sent
+ ******/
+
+function notify_via_pushover($message, $force = false) {
+	global $config;
+	if ((!isset($config['notifications']['pushover']['enabled']) && (!$force))
+	    || !$config['notifications']['pushover']['apikey']
+	    || !$config['notifications']['pushover']['userkey']) {
+		if ($force) {
```

Comment:
the main if checks for `!$force` so this condition will never be satisfied

Reply:
Can you take a second look? This is the same logic as the telegram notification. It works for me.

Reply:
You are right, I misread that